webapp/store/axiosErrorMiddleware.ts

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+
+import { applicationError } from '@webapp/components/error/actions'
+
+const createAxiosMiddleware =
+  (axios: any) =>
+  ({ dispatch }: any) => {
+    axios.interceptors.response.use(null, (error: any) => {
+      dispatch(applicationError(error))
+
+      return Promise.reject(error)
+    })
+
+    return (next: any) => (action: any) => next(action)
+  }
+
+export default createAxiosMiddleware(axios)
```

Comment:
can we have types instead of any?
maybe this can help: https://stackoverflow.com/questions/45339448/how-do-you-create-strongly-typed-redux-middleware-in-typescript-from-reduxs-typ ?